github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/trigger/check/NotUpdatedPRFilter.java

```diff
@@ -38,7 +38,9 @@ public boolean apply(GHPullRequest remotePR) {
         @CheckForNull GitHubPRPullRequest localPR = localRepo.getPulls().get(remotePR.getNumber());
 
         if (!isUpdated(remotePR, localPR)) { // light check
-            logger.getLogger().println(String.format("PR [#%s %s] not changed", remotePR.getNumber(), remotePR.getTitle()));
+            String logMessage = String.format("PR [#%s %s] not changed", remotePR.getNumber(), remotePR.getTitle());
+            logger.getLogger().println(logMessage);
+            LOGGER.info(logMessage);
```

Comment:
never use info in jenkins, that will spam system logger. If you want to debug special class add it in jenkins log handler with corresponding level. Here it should be trace or debug.

Reply:
Thanks, I changed it to `debug`